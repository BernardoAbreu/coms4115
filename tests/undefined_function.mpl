fun giveMe2() {
	return 2;
}

a = 4;
b = 7;

print( giveMe2() + sum(a, b) );