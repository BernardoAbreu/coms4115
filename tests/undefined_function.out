Fatal error: exception Failure("fun sum used but never declared")
